crates/biome_cli/src/logging.rs

```diff
@@ -40,7 +40,7 @@ pub fn setup_cli_subscriber(level: LoggingLevel, kind: LoggingKind) {
     };
 }
 
-#[derive(Debug, Default, Clone, Ord, PartialOrd, Eq, PartialEq)]
+#[derive(Copy, Debug, Default, Clone, Ord, PartialOrd, Eq, PartialEq)]
```

Comment:
We can revert the change. Not related to the PR.

Reply:
It happened from the small refactoring of `logging_level()` and `logging_kind()` in `commands/mod.rs`. It seemed a small enough DX improvement it wouldn’t require a separate PR 😅

Reply:
Fair. It wasn't mentioned in the PR description (did I miss it?)

Reply:
You’re right, should’ve mentioned it.

Reply:
Reverting after all, since it creates a lot of clippies elsewhere. Happy to make the change in a separate PR still :)